src/Form/UploadAvatarType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FrankProjects\UltimateWarfare\Form;
+
+use FrankProjects\UltimateWarfare\Entity\User;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Image;
+
+class UploadAvatarType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->setAction('/game/account/upload_avatar')
+            ->add(
+                'avatar',
+                FileType::class,
+                [
+                    'mapped' => false,
+                    'label' => 'label.avatar',
+                    'constraints' => [
+                        new Image([
```

Comment:
You probably need to add a use statement for this Image object? I get a ClassNotFoundException.

> Attempted to load class "Image" from namespace "Symfony\Component\Validator\Constraints"

Reply:
Maybe you missed that i've added the validator package in the composer.json file. A composer install might fix that problem for you.

Reply:
Ah missed that one! Forgot to composer install after installing the dependencies :)